rmmobility.com GA4 added and GA3 removed.

Raza403 opened the pull request:
This PR is made for https://github.com/fulldecent/mtssites/issues/4045

fulldecent reviewed (changes requested):
This removes a lot of features and documentation. 

Please explain and reference how cross-linking works for multiple domains under GA4. Please explain and reference how Analytics works for GA4. 

Also add back comment about when to include file.

Raza403 commented:
> This removes a lot of features and documentation.
> 
> Please explain and reference how cross-linking works for multiple domains under GA4. Please explain and reference how Analytics works for GA4.
> 
> Also add back comment about when to include file.

I read all the code and updated the changes.
We don't need to change our `trackingcode` file, our ID is updated in the `config` file. Updated the events, we don't need `email` and `phone number` events on this site, therefor I removed these. Removed `products.js` because we are no longer using it in our code.
